index.js

```diff
@@ -37,5 +39,9 @@ df.file = file => {
 		return Promise.reject(new Error('file required'));
 	}
 
+	if (!FS.existsSync(file)) {
```

Comment:
This would cause a race issue as the file might be deleted before it actually spawns `df`.